openedx/features/job_board/models.py

```diff
@@ -25,3 +25,6 @@ class Job(TimeStampedModel):
     contact_email = models.EmailField(max_length=255)
     logo = models.ImageField(upload_to='job-board/uploaded-logos/', blank=True, null=True)
 
+    def get_job_location(self):
+        return '{city}, {country}'.format(city=self.city, country=self.country.name)
```

Comment:
Please use @property for this, and rename it to `location` 
just like this example https://docs.djangoproject.com/en/1.11/topics/db/models/#model-methods is using @property for **full_name** 

https://docs.djangoproject.com/en/1.11/glossary/#term-property


in templates you will simply use `job.location`